components/returnless/actions/list-return-statuses/list-return-statuses.mjs

```diff
@@ -0,0 +1,32 @@
+import app from "../../returnless.app.mjs";
+
+export default {
+  key: "returnless-list-return-statuses",
+  name: "List Return Statuses",
+  description: "List all return statuses. [See the documentation](https://docs.returnless.com/docs/api-rest-reference/6129c8f41f66f-list-all-return-statuses)",
+  version: "0.0.2",
+  annotations: {
+    destructiveHint: false,
+    openWorldHint: true,
+    readOnlyHint: true,
+    idempotentHint: true,
```

Comment:
Are the `idempotentHint` annotations meant to be included?

Reply:
I'll remove it